virtual-setup/slack-procedures.md

```diff
@@ -1 +1,115 @@
+<p style="margin-top: 20px;"> </p>
+<p>Specifically, we use the <b>Cancer Data Science Slack</b> team administered by the CCDL.
+You can join Cancer Data Science Slack by following this link: [ccdatalab.org/slack](https://www.ccdatalab.org/slack)
+<br>
+<br>
+
+All Slack communications must adhere to our [code of conduct](https://github.com/AlexsLemonade/2020-may-training/blob/master/code-of-conduct.md).
+
+#### Instructions for setting up Slack
+
+We have instructions for setting up Slack for your operation system available.
+
+* [Linux](https://github.com/AlexsLemonade/training-modules/blob/master/virtual-setup/linux-instructions.md#slack)
+* [Mac](https://github.com/AlexsLemonade/training-modules/blob/master/virtual-setup/mac-instructions.md#slack)
+* [Windows](https://github.com/AlexsLemonade/training-modules/blob/master/virtual-setup/windows-instructions.md#slack)
+
+#### Table of Contents  
+
+- [How we use Slack during workshops](#how-we-use-slack-during-workshops)
+  - [Using the training-specific channel](#using-the-training-specific-channel)
+  - [Using Slack calling during training](#using-slack-calling-during-training)
+  - [Using direct messages during training](#using-direct-messages-during-training)
+- [General Slack use](#general-slack-use)
+    - [Attaching a file or image](#attaching-a-file-or-image)
+    - [Adding code blocks to messages](#adding-code-blocks-to-messages)
+
+## How we use Slack during workshops
+
+Course instructors will add you to a private channel specific to your training a few days before the start of your training workshop.
+
+If you are not added to the training specific channel by 3 days prior to the start of training or are having trouble getting started with Slack, please [direct message](#using-direct-messages-during-training) a CCDL staff member (`Chante Bethell`, `Candace Savonen`, `Josh Shapiro` or `Jaclyn N. Taroni`) in Cancer Data Science Slack or email [training@ccdatalab.org](mailto:training@ccdatalab.org).
+
+### Using the training-specific channel
+
+You can use the training-specific private channel to do the following:
+
+* Post questions during lectures
+* Post errors and get help with debugging
+``
+* Treat the channel as a study group – if you get stuck on an exercise notebook or your own data, post something so other participants and instructors can help you.
+
+If you have a question, it's very likely someone else in the course has the same question! 
+
+**Responses to questions should be in [threads](https://slack.com/help/articles/115000769927-Use-threads-to-organize-discussions-) as much as possible**, including any screenshots or images. 
+To start a thread, hover over the message you want to respond to and click the start a thread icon that appears on the top right of the message:
+
+<img src = screenshots/slack-start-a-thread.png width = 300>
```

Comment:
I think we should show it in the full context. i.e including the message.
ex:
<img width="459" alt="Screen Shot 2020-04-15 at 9 50 48 AM" src="https://user-images.githubusercontent.com/15315514/79345070-a009a100-7efe-11ea-962d-dad3510e6f0c.png">